.circleci/config.yml

```diff
@@ -14,17 +14,19 @@ workflows:
           attach-workspace: true
           image: foodlestech/logzio-docker-collector-metrics
           tag: develop
+          dockerfile: dockerfile
           filters:
             branches:
               only:
-                - develop
+                - fix/ci_build
```

Comment:
```suggestion
                - develop
```
❓

Reply:
I pushed a test commit to see if it builds well